classes/helper/PowatagTotAdminTabHelper.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+* 2007-2015 PrestaShop.
+*
+* NOTICE OF LICENSE
+*
+* This source file is subject to the Open Software License (OSL 3.0)
+* that is bundled with this package in the file LICENSE.txt.
+* It is also available through the world-wide-web at this URL:
+* http://opensource.org/licenses/osl-3.0.php
+* If you did not receive a copy of the license and are unable to
+* obtain it through the world-wide-web, please send an email
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+*  @author    PrestaShop SA <contact@prestashop.com>
+*  @copyright 2007-2014 PrestaShop SA
+*
+*  @version  Release: $Revision: 7776 $
+*
+*  @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+*  International Registered Trademark & Property of PrestaShop SA
+*/
+
+class PowatagTotAdminTabHelper
+{
+    /**
+     * Function to delete admin tabs from a menu with the module name.
+     *
+     * @param string $name name of the module to delete
+     */
+    public static function deleteAdminTabs($name)
+    {
+        Db::getInstance()->Execute('DELETE FROM '._DB_PREFIX_.'tab_lang WHERE id_tab IN (SELECT id_tab FROM '._DB_PREFIX_.'tab WHERE module = "'.pSQL($name).'")');
```

Comment:
You should use the PrestaShop Object Model (in your case Tab()) to perform action rathan than communicate directly with the database. Use directly the tab delete action (that will carry multishop).